examples/simple-tcpudp/main.go

```diff
@@ -0,0 +1,196 @@
+// Copyright 2019 Google LLC All Rights Reserved.
```

Comment:
You might hate me for this - but does it make sense to pass a flag to https://github.com/googleforgames/agones/blob/master/examples/simple-udp/main.go and when enabled, turn on a TCP listener as well?

Rather than going to all the trouble to create a whole new exampe?